src/libraries/Native/Unix/System.IO.Ports.Native/pal_termios.c

```diff
@@ -495,7 +511,11 @@ int32_t SystemIoPortsNative_TermiosReset(intptr_t handle, int32_t speed, int32_t
             return -1;
         }
 
+#if HAVE_CFSETSPEED
```

Comment:
same as above. `cfsetspeed() is a 4.4BSD extension.` We may use cfsetispeed and cfsetospeed on all platforms.